Landing page banner

RuthieRuth opened the pull request:
Landing page banner, not really working will continue

Pujakumari1511 commented on frontend/src/LandingPage.tsx:
```diff
@@ -0,0 +1,189 @@
+import image2 from "./images/image2.png";
+import image3 from "./images/image3.png";
+import image4 from "./images/image4.png";
+//import { AnimatePresence, motion } from "framer-motion";
```
remove commented lines

Pujakumari1511 commented on frontend/src/LandingPage.tsx:
```diff
@@ -0,0 +1,189 @@
+
+        {/* Image Transition */}
+        <Box position="relative" width="50%" height="200px" overflow="hidden">
+        {/* <AnimatePresence>
+      <motion.img
+        key={currentImage}
+        src={images[currentImage]}
+        alt="Slideshow Image"
+        initial={{ opacity: 0 }}
+        animate={{ opacity: 1 }}
+        exit={{ opacity: 0 }}
+        transition={{ duration: 1 }}
+        style={{
+          width: "100%",
+          height: "100%",
+          objectFit: "contain",
+          position: "absolute",
+        }}
+      />
+      </AnimatePresence> */}
+
+      {/* Overlaying Text */}
```
remove unused code

Pujakumari1511 commented on backend/src/index.ts:
```diff
@@ -1,12 +1,29 @@
-import express from 'express';
-// import cors from 'cors';
+import express from "express";
```
if there is no changes in this file revert it to main as Nikita is already working on this in his PR, this file will create unnecerssary conflict

RuthieRuth commented:
- Moved theme mode to header.
- Removed routing parts

![Screenshot 2025-03-31 at 14 22 34](https://github.com/user-attachments/assets/c5b99d90-efb0-4bfd-94f7-119f297ffd09)
![Screenshot 2025-03-31 at 14 23 03](https://github.com/user-attachments/assets/17a85266-b7ce-4a92-954a-d6e73f9999c2)

SneRam-0105 reviewed (approved):
Landing page is having the themes now and okay to be approved.
